texttospeech/ssml_addresses/tts.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2019 Google LLC
+
```

Comment:
Can delete these empty lines in the License, but keep the ones that have the `#` in them.